aws/resource_aws_ecr_repository.go

```diff
@@ -199,6 +256,12 @@ func resourceAwsEcrRepositoryUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	if d.HasChange("encryption_configuration") {
+		if err := resourceAwsEcrRepositoryUpdateImageScanningConfiguration(conn, d); err != nil {
+			return err
+		}
+	}
+
```

Comment:
All the encryption configuration attributes are marked `ForceNew` so this should be removed.

```suggestion
```